src/translations/en.ts

```diff
@@ -1598,6 +1598,7 @@ export default {
 		"yourCalendars_label": "Your calendars",
 		"yourFolders_action": "YOUR FOLDERS",
 		"yourMessage_label": "Your message",
-		"you_label": "You"
+		"you_label": "You",
+		"eventNoLongerExists_msg": "The requested event no longer exists"
```

Comment:
I'd rather say "the edited event no longer exists and couldn't be updated" to be very explicit